README.md

```diff
@@ -12,11 +12,11 @@ VsCode.
 
 Saluma Hasan
 
-GitHub: @githubhandle
+GitHub: @saluumaa
 
-Twitter: @twitterhandle
+Twitter: @SalmaHIbrahim20
 
-LinkedIn: LinkedIn
+LinkedIn: salma ibrahim
```

Comment:
- [ ] **Good job on making some of the Requested changes 👍 👍**
    However, It would be better to use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections present in yours and placeholders are replaced with your project and personal data As Suggested by the previous Reviewer 🙏